src/components/v5/shared/ProgressBar/ProgressBar.tsx

```diff
@@ -23,10 +24,14 @@ const ProgressBar: FC<ProgressBarProps> = ({
         })}
       >
         <span
-          className={clsx('bg-blue-400 h-full absolute left-0 top-0', {
-            'rounded-lg': isTall,
-            rounded: !isTall,
-          })}
+          className={clsx(
+            className,
```

Comment:
If you're adding `className` prop, then usually it means that it's added to the wrapper and not to the element inside. Please change the naming or move this prop to the top.